### PR TITLE
Migrate simplified datamodel classes

### PR DESCRIPTION
**Pull request by HoodedCrow**
All server-side wiring removed.

### CODE REVIEW

File: src/main/java/org/b612foundation/adam/datamodel/PropagatorConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    OMIT,
+    /** Treat the body as point-mass. */
+    POINT_MASS,
+    /** Use a spherical harmonics model. Assumes there is a default model. Not always applicable. */
+    SPHERICAL_HARMONICS
+  }
+
+  /** Globally unique id of this config. */
+  private String uuid;
+  /** Human-readable description. Not used in hash code or equal. */
```
Comment (laurahlark) [inline src/main/java/org/b612foundation/adam/datamodel/PropagatorConfiguration.java]: It does seem to be used in hash code and equal. Not sure what behavior should be.

File: src/main/java/org/b612foundation/adam/datamodel/RunDescription.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
+package org.b612foundation.adam.datamodel;
+
+import java.util.Objects;
+
+import org.b612foundation.adam.opm.OrbitParameterMessage;
```
Comment (laurahlark) [inline src/main/java/org/b612foundation/adam/datamodel/RunDescription.java]: I suppose at some point we'll pull this in? Alternatively we could keep opms as strings in Run objects to mirror Batch objects.

Comment (laurahlark) [reply src/main/java/org/b612foundation/adam/datamodel/RunDescription.java]: Oh wait, obviously that's already here, pulled over with the other opm stuff. Nevermind!

File: src/main/java/org/b612foundation/adam/datamodel/PropagatorConfiguration.java (hunk 1 above)
Comment (HoodedCrow) [reply src/main/java/org/b612foundation/adam/datamodel/PropagatorConfiguration.java]: Hmmm, not sure when it snuck in. Removed from `equals` and `hashCode`

**Comment by HoodedCrow** [comment]
Readme is technically already there (with a title). Agreed on need for compilation instructions, especially since Components-dependent code will be migrating here as well.

**Pull request merged**